src/seedu/addressbook/logic/Logic.java

```diff
@@ -85,7 +87,9 @@ public CommandResult execute(String userCommandText) throws Exception {
     private CommandResult execute(Command command) throws Exception {
```

Comment:
Remember to update with header comments, now that the behavior of `execute(Command command) `has changed.

Reply:
Alright I will do so in the upcoming tasks. Thank you.